### ISSUE TITLE
Conbee 2 - updates

### ISSUE THREAD

**Issue by TungstenE2** [dresden-elektronik/deconz-rest-plugin#2767]
Hi all,

According to gateway menu my version 2.05.71is up to date.
I thought there is OTA update available. 

![grafik](https://user-images.githubusercontent.com/39955685/81852767-d6692900-955b-11ea-926c-531ae20bc2ab.png)

But now I see manual instructions and several versions.

https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually

http://deconz.dresden-elektronik.de/deconz-firmware/?C=M;O=D

![grafik](https://user-images.githubusercontent.com/39955685/81853126-57c0bb80-955c-11ea-906d-640a5cd8e17b.png)

Which version is latest stable to be used with Conbee 2 on Raspberry Pi?

deCONZ_ConBeeII_0x26580700.bin.GCF
deCONZ_Rpi_0x26350500.bin.GCF

I am a bit confused about the updates. 

thx

**Comment by Gizmo-Ger** [comment]
Those are the firmwares for several zigbee devices. Depending on which device you are using. In your case it´s a ConbeeII, an USB zigbee stick. The other one is an addon module for a Raspberry. Latest release for Phoscon is V2_05_75.

**Comment by SwoopX** [comment]
> Latest release for Phoscon is V2_05_75.

That's the release fo deCONZ (which includes Phoscon).

The reason why the newer firmware versions are not suggested as updates is that they're still tagged as beta. However, this is expected to change soon.

**Comment by TungstenE2** [comment]
thx for the info.

But I see V2_05_74_stable, which is also not shown in my Phoscon. Any idea why?

https://github.com/dresden-elektronik/deconz-rest-plugin/releases

**Comment by TungstenE2** [comment]
just updated to  V2_05_74, but why was this update not listed in Phoscon app?

**Comment by SwoopX** [comment]
> 
> 
> just updated to V2_05_74, but why was this update not listed in Phoscon app?

You meant to V2_05_75, right? Presumably, .75 is still tagged as beta. When you use deconz' repository via APT, it depends on which channel you've chosen as source, beta or stable.

**Comment by TungstenE2** [comment]
As stated I updated to V2_05_74 as I subscribed to main.

My question was: while beeing on V2_05_71 the app sated I am on the latest version, while V2_05_74 was already available. So there is a bug in the info displayed regarding the latest version.

**Comment by SwoopX** [comment]
ok, clear now. I'm not too sure though if the update notification is just for the firmware, deconz or both. If it's just firmware, it would be indeed beneficial to extend that to deconz version as well.

**Comment by djashjones** [comment]
I thought there was going to be a major update this week?

**Comment by danli003** [comment]
Yes, there hasn't been much updates lately. Does anyone have any info on what is going on?

**Comment by Mimiix** [comment]
This is a phoscon related question. Moving it to their git.
